hack/jenkins/e2e-image/Dockerfile

```diff
@@ -0,0 +1,50 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This file creates a build environment for building and running kubernetes
+# unit and integration tests
+
+FROM gcr.io/google-containers/kubekins-test:v20160810
+MAINTAINER  Erick Fejta <fejta@google.com>
+
+# Defaults of all e2e runs
+ENV E2E_UP=true \
+    E2E_TEST=true \
+    E2E_DOWN=true
+
+# Customize these as appropriate
+ENV E2E_PUBLISH_GREEN_VERSION=false \
+    GINKGO_PARALLEL_NODES=25 \
```

Comment:
Just reverting `hack/jenkins/dockerized-e2e-runner.sh` might be enough, but a full revert is probably ok too.